src/webots/nodes/WbBaseNode.cpp

```diff
@@ -16,6 +16,7 @@
 #include "WbBasicJoint.hpp"
 #include "WbBoundingSphere.hpp"
 #include "WbDictionary.hpp"
+#include "WbField.hpp"
```

Comment:
Not needed.
```suggestion
```

Reply:
Yes, it is needed by the connect instruction.